specutils/analysis/flux.py

```diff
@@ -116,3 +124,83 @@ def _compute_equivalent_width(spectrum, continuum=1, regions=None):
     ew =  dx - (line_flux / continuum)
 
     return ew.to(calc_spectrum.spectral_axis.unit)
+
+def is_continuum_below_threshold(spectrum, threshold=0.01):
+    """
+    Determine if the baseline of this spectrum is less than the threshold.
+    I.e., an estimate of whether or not the continuum has been subtracted.
+
+    If the threshold is an `~astropy.units.Quantity` then compare the median
+    of the flux to the threshold.
+
+    If the threshold is a float then the spectrum's uncertainty will be
+    used or an estimate of the uncertainty. If the uncertainty is present then the
+    threshold is compared to the median of the flux divided by the
+    uncertainty.  If the uncertainty is not present then the threshold
+    is compared to the median of the flux divided by the
+    `~astropy.stats.median_absolute_deviation`.
+
+    Parameters
+    ----------
+    spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
+        The spectrum object over which the width will be calculated.
+
+    threshold: float or `~astropy.units.Quantity`
+        The tolerance on the quantification to confirm the continuum is
+        near zero.
+
+    Returns
+    -------
+    is_continuum_below_threshold: bool
+        Return True if the continuum of the spectrum is below the threshold, False otherwise.
+
+    """
+
+    flux = spectrum.flux
+    uncertainty = spectrum.uncertainty if hasattr(spectrum, 'uncertainty') else None
+
+    # Apply the mask if it exists.
+    if hasattr(spectrum, 'mask') and spectrum.mask is not None:
+        flux = flux[~spectrum.mask]
+        uncertainty = uncertainty[~spectrum.mask] if uncertainty else uncertainty
+
+    # If the threshold has units then the assumption is that we want
+    # to compare the median of the flux regardless of the
+    # existence of the uncertainty.
+    if hasattr(threshold, 'unit'):
```

Comment:
I think we want to treat *dimensionless* quantities as though they are floats.  So something like:
```suggestion
    if hasattr(threshold, 'unit') and not threshold.unit == u.dimensionless_unscaled:
```
?